### PR TITLE
Update documenation following Python3 support in the agent

### PR DESCRIPTION
**Pull request by hush-hush**
### What does this PR do?

Update documenation following Python3 support in the agent

### CODE REVIEW

**Review by apigirl — DISMISSED** [review]
docs look good

File: README.md [hunk 1]
```diff
@@ -46,8 +46,20 @@ To start working on the Agent, you can build the `master` branch:
 2. cd into the project folder: `cd $GOPATH/src/github.com/DataDog/datadog-agent`.
 3. Install project's dependencies: `invoke deps`.
    Make sure that `$GOPATH/bin` is in your `$PATH` otherwise this step might fail.
-4. Build the `rtloader` dependency with `invoke rtloader.build && invoke rtloader.install`. You will need CMake installed and a C++ compiler for this to work.
-5. Build the agent with `invoke agent.build --build-exclude=systemd`
+4. Build the `rtloader` dependency with `invoke rtloader.build && invoke
+   rtloader.install`. You will need CMake installed and a C++ compiler for this
+   to work.
+   `rtloader` is in charge of loading and running Python. By default `rtloader`
+   will be build for Python2 you can choose which versions of Python you want
```
Comment (albertvaka) [inline README.md]:
```suggestion
   will be built for Python2, but you can choose which versions of Python you want
```

File: README.md [hunk 2]
```diff
@@ -46,8 +46,20 @@ To start working on the Agent, you can build the `master` branch:
...
+   to support:
+   - `invoke rtloader.build -p 2` for Python2 only
+   - `invoke rtloader.build -p 3` for Python3 only
+   - `invoke rtloader.build -p 2,3` for both Python2 and Python3
+5. Build the agent with `invoke agent.build --build-exclude=systemd`. You can
+   specify custome Python location for the agent (useful when using
```
Comment (albertvaka) [inline README.md]:
```suggestion
   specify a custom Python location for the agent (useful when using
```

**Review by albertvaka — DISMISSED** [review]
Found some typos but content changes LGTM.

**Pull request merged**